realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -244,6 +246,7 @@ public Builder(Context context) {
                 throw new IllegalArgumentException("A non-null Context must be provided");
             }
             initializeBuilder(context.getFilesDir());
+            ReLinker.loadLibrary(context, "realm-jni");
```

Comment:
The problem with the static block is that it doesn't have access to a `Context`.